apps/tsilhqotin-language-hub/src/Components/Pages/Home/Home.tsx

```diff
@@ -1,20 +1,22 @@
-import { Button, Typography } from '@mui/material';
+import LaunchIcon from '@mui/icons-material/Launch';
+import ShopIcon from '@mui/icons-material/Shop';
+import { Card, CardContent, Typography } from '@mui/material';
 import { Link } from 'react-router-dom';
 import ScrollToDiv from '../../Widgets/ScrollButton/ScrollDiv';
 import ScrollToTop from '../../Widgets/ScrollButton/ScrollToTop';
 import './Home.module.css';
+import './Home.scss';
 
 function App(): JSX.Element {
     return (
         <Typography className="page" component={'span'} variant={'body2'}>
             <div className="homeBackground">
                 <ScrollToTop />
-
                 <div className="topDiv">
                     <div className="backdrop">
                         <div className="title">
                             <div>
-                                <b>Tŝilhqot’in Ch’ih Yaltɨg</b>
+                                <b className="siteTitle">Tŝilhqot’in Ch’ih Yaltɨg</b>
```

Comment:
Pull the site title, etc., from a config.

Reply:
Although this is probably something to worry about in the future. As it stands, this is kind of a one-off.